scripts/workers/audit-arch-run.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+# audit-arch-run.sh — 단발성. launchd가 주기적 호출.
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_DIR="/Users/mark/workspace/minglit"
+WORKTREE_DIR="/Users/mark/workspace/minglit-workers/audit-arch"
+PROMPT_FILE="$SCRIPT_DIR/prompts/audit-arch.txt"
+SESSION_TIMEOUT=1800
+REPO="Mark-Yun/minglit"
```

Shellcheck (0.11.0)

[warning] REPO appears unused. Verify use (or export if used externally).

(SC2034)